.github/workflows/ci.yml

```diff
@@ -2,30 +2,52 @@ name: CI
 on:
   push:
     branches:
-      - '**'
-      - '!dependabot/**'
-  pull_request:
-    # NOTE: does not work for some reasons
-    # branches:
-    #   - 'dependabot/**'
+      - 'main'
+  pull_request_target:
 
 env:
   STRIPE_PUBLISHABLE_KEY: ${{ secrets.TEST_STRIPE_PUBLISHABLE_KEY }}
   STRIPE_SECRET_KEY: ${{ secrets.TEST_STRIPE_SECRET_KEY }}
   COMPOSE_FILE: docker-compose.yml:docker-compose.playwright.yml
+  MAINTAINERS: |
```

Comment:
Can we automate getting this list?

Reply:
I create the issue to take over it.
https://github.com/stripe-samples/accept-a-payment/issues/1773